packages/manager/modules/bm-server-components/src/edit-name/template.html

```diff
@@ -0,0 +1,31 @@
+<form
+    id="bmServerEditName"
+    name="bmServerEditName"
+    data-ng-submit="bmServerEditName.$valid && $ctrl.saveName()"
+    novalidate
+>
+    <oui-modal
+        data-heading="{{:: 'server_name_edit_title' | translate }}"
+        data-primary-label="{{:: 'server_name_edit_common_save' | translate }}"
+        data-secondary-label="{{:: 'server_name_edit_common_cancel' | translate }}"
+        data-secondary-action="$ctrl.handleCancel()"
+        data-on-dismiss="$ctrl.handleCancel()"
+        data-loading="$ctrl.isUpdating"
+        data-type="warning"
+    >
+        <p data-translate="server_name_edit_help"></p>
+        <oui-field
+            data-label="{{:: 'server_name_edit_displayname_label' | translate }}"
+        >
+            <input
+                class="oui-input"
+                type="text"
+                name="bm-server-edit-name"
+                id="bm-server-edit-name"
+                data-ng-model="$ctrl.updatedName"
+                required
+                autofocus
```

Comment:
`maxlength="255"` and validation if any

Reply:
This will be added to oui-input component. Once its added, it will be applicable to all input fields.

Reply:
`oui-input` is a class so I'm not sure how  the `maxlength` attribute can be applied

Reply:
we are planning to provide a wrapper component for oui-input with default length and special character validation